target/linux/ramips/image/mt7621.mk

```diff
@@ -88,6 +88,13 @@ define Device/hc5962
 endef
 TARGET_DEVICES += hc5962
 
+define Device/k2p
+  DTS := K2P
+  IMAGE_SIZE := 16056320
```

Comment:
However I found that the variable ralink_default_fw_size_16M is 16121856(in target/linux/ramips/image/Makefile) which isn't equal to the partition size of firmware partition in the generic partition table of Ralink SDK(0x30000 for u-boot,0x10000 for u-boot-env 0x10000 for factory and 0xfb0000(or 16449536 bytes) for firmware).